capsul/pipeline/pipeline.py

```diff
@@ -61,11 +59,7 @@ def pipeline_definition(self):
               self.add_link('proc2.out1->main_switch.in2_switch_out1')
               self.add_link('proc2.out1->main_switch.in2_switch_out2')
 
-    After execution of :py:meth:`pipeline_definition`, the inner nodes
-    parameters which are not connected will be automatically exported to the
-    parent pipeline, with names prefixed with their process name, unless they
-    are listed in a special "do_not_export" list (passed to
-    :py:meth:`add_process` or stored in the pipeline instance).
+    After execution of :py:meth:`pipeline_definition`, the inner nodes parameters which are not connected will be automatically exported to the parent pipeline, with names prefixed with their process name, unless they are listed in a special "do_not_export" list (passed to :py:meth:`add_process` or stored in the pipeline instance).
```

Comment:
I think I am missing something here. This change (and other similar changes) is the contrary of PEP 8, no ?